test/test-helpers.js

```diff
@@ -312,6 +313,35 @@ const TestHelpers = {
         return this.makePlayer(tag, opts);
     },
 
+    makeAudioScreenPlayer(newOptions) {
+        // use polyfill in Firefox for now, see:
```

Comment:
i guess this polyfill statement was cop/pastad, maybe it should be refactored into a new method or something.

Reply:
723be4f